packages/salesforcedx-vscode-core/src/commands/isvdebugging/bootstrapCmd.ts

```diff
@@ -179,16 +178,21 @@ export class IsvDebugBootstrapExecutor extends SfdxCommandletExecutor<{}> {
 
   public buildRetrievePackagesSourceCommand(
     data: IsvDebugBootstrapConfig,
-    packageNames: string[]
+    packageName: string
   ): Command {
     return new SfdxCommandBuilder()
       .withDescription(
-        nls.localize('isv_debug_bootstrap_step6_retrieve_packages_source')
+        nls.localize(
+          'isv_debug_bootstrap_step5_retrieve_packages_source',
```

Comment:
nit: step# in localization keys leads to mislabeled keys.  You can take the step# out of the label keys if so inclined.

Reply:
Thanks for the suggestion! i will make that change.